lib/ae_mdw/names.ex

```diff
@@ -1018,4 +1034,21 @@ defmodule AeMdw.Names do
       tx: :aens_update_tx.for_client(name_update_tx)
     }
   end
+
+  defp count_active_names(state, nil) do
+    state
+    |> Collection.stream(Model.AccountNamesCount, :forward, nil, nil)
+    |> Enum.reduce(0, fn owner, acc ->
+      count_active_names(state, owner) + acc
+    end)
```

Comment:
I thought we weren't going to traverse through an unlimited list of results

Reply:
The list is very short, so i thought it would be ok. There's only one record per account

Reply:
We never go through unlimited lists, this wouldn't be a proper implementation. For total active names we can simply take the last stat from the database:
```elixir
defp count_active_names(state, nil) do
  case State.prev(state, Model.TotalStat, nil) do
    {:ok, Model.total_state(active_names: active_names) -> active_names
    :none -> 0
  end
end
```